rdmo/projects/templates/projects/project_detail_sidebar_parent_import.html

```diff
@@ -6,7 +6,7 @@
     <input type="hidden" name="method" value="import_project">
 
     <select class="form-control" name="source">
-        {% for project in project.get_ancestors %}
+        {% for project in ancestors_import %}
```

Comment:
```
{% has_perm 'projects.view_project_object' request.user project as can_view_project %}
{% if can_view_project %}
```

same as above. We probably need to add some `prefetch_related/select_related` to the database query.